src/output/writeCGNSSurface.F90

```diff
@@ -186,7 +186,7 @@ subroutine writeCGNSSurfaceSol(famList)
              if (ierr /= CG_OK) &
                   call terminate("WriteCGNSSurfaceSol", &
                   "Something wrong when calling cg_base_write_f for &
-                  isoSurface")
+                  &isoSurface")
```

Comment:
Some of these changes would be better if we just removed the line break. I can change these in a future PR though

Reply:
Yeah, there are more of these. I can update this one here, as this has already been touched.